### DISCUSSION TITLE
CORS error after updating to version 9.0.0-rc52

### DISCUSSION THREAD

**Discussion by sagerio** [directus/directus#4737] [Archive]
After upgrading the database and the js-sdk to the lastest, out of nowhere I get an CORS-Error:

`Access to XMLHttpRequest at '...' from origin 'http://localhost:4200' has been blocked by CORS policy: Request header field accept-language is not allowed by Access-Control-Allow-Headers in preflight response.`

Any ideas how to fix this?

Thanks & regards

**Comment by rijkvanzanten** [comment]
Nothing changed in the CORS setup in this version. Are you sure it's caused by the upgrade itself, and not an external factor?

**Reply by sagerio** [reply]
Hmmm, it worked like a charm until I updated a few minutes ago...
But I'll investigate and close this in case I find an external cause.

**Reply by rijkvanzanten** [reply]
Weird! Lemme know if you find anything. There's always the chance that it's a bug in Directus of course 🙂

**Reply by sagerio** [reply]
OK, I tested in another browser (Chrome) instead of the default one (Edge Chromium) and there was no error,
So I restarted everything and now all is fine.
Looks like some kind of hickup in a long-running instance :-)
Thanks a lot, will close this.
